### ISSUE TITLE
High Contrast versions of light/dark mode

### ISSUE THREAD

**Issue by TheJaredWilcurt** [vue-dev-tools-accessibility/v0#22]
Add a checkbox to the settings page for High Contrast mode.

It should pass WCAG AAA and APCA Silver+. Borders should have contrast too.
